nixos/tests/plikd.nix

```diff
@@ -0,0 +1,17 @@
+import ./make-test-python.nix ({ lib, ... }: {
+  name = "plikd";
+  meta = with lib.maintainers; {
+    maintainers = [ freezeboy ];
+  };
+
+  machine = { ... }: let
+  in {
+    services.plikd.enable = true;
+  };
+
+  testScript = ''
```

Comment:
Bonus points if you upload and download a file.

Reply:
challenge accecpted